lessonbuilder/tool/bin/src/resources/messages_fa_IR.properties

```diff
@@ -600,7 +600,6 @@ simplepage.context-link-title-2=\u0646\u0645\u0627\u06cc\u0634 \u0645\u062a\u064
 
 simplepage.student-comments-grading=\u0646\u0638\u0631\u0627\u062a \u0628\u0631\u0648\u06cc \u0645\u062d\u062a\u0648\u0627\u06cc \u062f\u0627\u0646\u0634 \u0622\u0645\u0648\u0632 \u0628\u0631\u0627\u06cc "{}"
 simplepage.comments-grading=\u0646\u0638\u0631\u0627\u062a \u0628\u0631\u0627\u06cc "{}"
-simplepage.question-grading=\u067e\u0627\u0633\u062e \u0628\u0647 "{}"
```

Comment:
is this path correct bin/src/resources? should it be remove?

Reply:
done. removed